src/framecontainer/viewerConfig.ts

```diff
@@ -53,13 +53,20 @@ export const VIEWER_PARAMETERS: ViewerParameters = {
   p2r: false,
   horizontalScrolling: false,
   storage: true,
-  development: false,
+  development: true,
```

Comment:
should this be defaulting to true? or did you mistakenly check this in when you changed it for your developing purposes?